### PR TITLE
018 patch/food items is active

### PR DESCRIPTION
**Pull request by usuajiayako**

### CODE REVIEW

File: __test__/app.test.js [hunk 1]
```diff
@@ -61,7 +62,27 @@ describe('/api', () => {
...
-            course: 'dessert'
+            course: 'dessert',
+            is_active: true,
+          });
+        });
+    });
+    it('SUCCESS status 201 - changes active state of a food item', () => {
+      return request(app)
+        .patch('/api/food-items/1')
+        .send({ food_item_id: 1,
```
Comment (joemould) [inline __test__/app.test.js:73]: do we need to send the full food item if just patching the active state?

File: controllers/errors.controllers.js [hunk 2]
```diff
@@ -25,5 +25,6 @@ exports.handlePSQLErrors = (err, req, res, next) => {
 };
 
 exports.handleServerErrors = (err, req, res, next) => {
+  console.log(err)
```
Comment (Zakariyya1) [inline controllers/errors.controllers.js:28]: We don't need this console log now

File: __test__/app.test.js [hunk 3]
```diff
@@ -61,7 +62,27 @@ describe('/api', () => {
             food_item_id: 1,
             name: 'New York Cheesecake',
             price: 4.5,
-            course: 'dessert'
+            course: 'dessert',
+            is_active: true,
+          });
+        });
+    });
+    it('SUCCESS status 201 - changes active state of a food item', () => {
```
Comment (joemould) [inline __test__/app.test.js:70]: I was told on backend nc news that PATCH is a 200 status code when successful

File: controllers/food-items.controllers.js [hunk 4]
```diff
@@ -8,7 +8,7 @@ const {
 exports.getFoodItems = (req, res, next) => {
   selectFoodItems().then(foodItems => {
     res.send({ foodItems });
-  });
+  }).catch((err) => {console.log(err)})
```
Comment (Zakariyya1) [inline controllers/food-items.controllers.js:11]: This can be changed to .catch(next).

File: controllers/errors.controllers.js (hunk 2 above)
Comment (joemould) [inline controllers/errors.controllers.js:28]: remove unnecessary console logs

**Review by Zakariyya1 — COMMENTED** [review]
A couple of console logs to remove, otherwise great!

File: __test__/app.test.js (hunk 1 above)
Comment (usuajiayako) [reply __test__/app.test.js:73]: we could just send the is_active state and it's a matter of preference for the frontend. What I had in mind was a user can see the all the details of food item and change where necessary and send request to backend.

File: controllers/errors.controllers.js (hunk 2 above)
Comment (usuajiayako) [reply controllers/errors.controllers.js:28]: Yeah, I can delete that on github.

Comment (usuajiayako) [reply controllers/errors.controllers.js:28]: Yeah, I can delete that on github.

File: controllers/food-items.controllers.js (hunk 4 above)
Comment (usuajiayako) [reply controllers/food-items.controllers.js:11]: I'll change this one

File: __test__/app.test.js (hunk 3 above)
Comment (usuajiayako) [reply __test__/app.test.js:70]: change that to 200

**Review by Zakariyya1 — APPROVED** [review]

File: __test__/app.test.js (hunk 1 above)
Comment (joemould) [reply __test__/app.test.js:73]: yeah I can see what you mean, food for thought for my patch request as well!

**Review by joemould — APPROVED** [review]

**Pull request merged**